data/json/scenarios/scenarios.json

```diff
@@ -190,10 +190,10 @@
     "ident": "heli_crash",
     "name": "Helicopter Crash",
     "points" : -2,
-    "description": "You've crashed your precious helicopter, it'll never fly again.  Monsters don't bother you, life without your Loach is worthless anyway.",
+    "description": "You've crashed your precious helicopter, it'll never fly again.  You're the only survivor, but life without your Loach is worthless anyway.",
```

Comment:
The new helicopters are a UH-60, V-22, and AH-64. It's a minor thing, but it probably wouldn't hurt to replace the Loach reference (which is an OH-6) with something else.

Reply:
What is 'Loach' a reference to?  In the Heli-crash handler in game.cpp, I rename the helicopter to "Loach Wreckage".

Reply:
As noted in the comment above, it's an OH-6. See more [here](https://en.wikipedia.org/wiki/Hughes_OH-6_Cayuse). I think you'd be better off leaving the names from the json (they're fun for the verisimilitude), and making the description above more generic, e.g. `You're the only survivor, but life without your bird is worthless anyway.`

Reply:
> What is 'Loach' a reference to? In the Heli-crash handler in game.cpp, I rename the helicopter wreck that the player is moved in to to "Loach Wreckage".

Loach is supposed to be the tender nickname for the helicopter its pilot used to name it.